dataland-frontend/src/components/pages/ViewLksg.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <ViewFrameworkBase :companyID="companyID" dataType="lksg" @updateDataId="receiveDataId">
+    <template v-if="frameworkDataId">
+      <div class="grid">
+        <div class="col-12">
+          <LksgPanel :dataID="frameworkDataId" />
+        </div>
+      </div>
+    </template>
+    <div v-if="frameworkDataId === null" class="col-12 text-left">
+      <h2>No LKSG data</h2>
+    </div>
+  </ViewFrameworkBase>
+</template>
+
+<script lang="ts">
+import ViewFrameworkBase from "@/components/generics/ViewFrameworkBase.vue";
+import LksgPanel from "@/components/resources/frameworkDataSearch/lksg/LksgPanel.vue";
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  name: "ViewLksg",
+  components: { ViewFrameworkBase, LksgPanel },
+  props: {
+    companyID: {
+      type: String,
+    },
+  },
+  data() {
+    return {
+      frameworkDataId: [] as string[],
+    };
+  },
+  methods: {
+    receiveDataId(id: string | []) {
+      let dataIdsArray = [] as string[];
+      if (Object.prototype.toString.call(id) === "[object String]") {
```

Comment:
I am not sure for what this distinction is needed. Could you briefly explain?

Reply:
because if there is only one year in the data, we get a string with dataID if there is more than one, we get arrays ['string', 'string']. I would like to have the same data type, so I convert it to array with one or more elements

Reply:
Maybe we should change the endpoint so it always returns and array of data. So even if there is one it returns ['string']. Would be more consistent from my point of view.

Reply:
changed